tools/distrib/python/grpcio_tools/grpc_tools/protoc.py

```diff
@@ -63,9 +67,20 @@ def _maybe_install_proto_finders():
                         ),
                     ]
                 )
-                sys.path.append(
-                    pkg_resources.resource_filename("grpc_tools", "_proto")
-                )
+
```

Comment:
I think this might be a bit more understandable

```python
def get_package_path(segments: List[str]) -> str:
  if sys.version_info ...
    pass
  else:
    pass

...
sys.path.append(get_package_path(("grpc_tools", "_proto")))
```